src/style.css

```
@@ -0,0 +1,45 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+
+}
+
+.stars__star + .stars__star {
+  margin-left: 4px;
+}
+
+.stars--1 > .stars__star:first-child {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--2 > .stars__star:nth-child(-n+2) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
```

```suggestion
.stars--1 > .stars__star:first-child,
.stars--2 > .stars__star:nth-child(-n+2) {
  background-image: url(images/star-active.svg);
}
```
change it for all stars--
and you don`t need background-repeat: no-repeat; here